### PR TITLE
adding npm scripts for compiling ES5 dist through babel

### PR DESCRIPTION
**Pull request by criography**
This PR adds a build script that would allow to use this library in opinionated environments ESNext is not fully supported, like [create-react-app](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#npm-run-build-fails-to-minify).

It doesn't alter the source at all.

### CODE REVIEW

**Comment by onury** [comment]
Thanks @criography. I'd like to merge this but 2 points:
- Should better make it universal module at once (via webpack maybe) so it'd work directly in the browser too.
- We might still have a `/dist` dir but import would be prettier like: 
`import { invert } from 'invert-color'` 

Would you mind updating your PR for this?

**Comment by criography** [comment]
1. It was simpler to just use babel's plugin to achieve UMD.
2. I wasn't sure if this is what you meant, but if the path is not specified, both `require` and `import` methods will use whatever is set as `main` declaration in `package.json`. At least that's my understanding - I don't have much experience in this field. 
In this commit, I set `main` to `dist` output, which seems to be the common strategy for libraries out there. That said, I'm open to suggestions how to handle it differently.

File: package.json [hunk 1]
```diff
@@ -2,11 +2,11 @@
   "name": "invert-color",
   "version": "1.1.0",
   "description": "Generates inverted (opposite) version of the given color.",
-  "main": "src/index.js",
+  "main": "dist/index.js",
   "scripts": {
     "test": "node test",
-    "build": "babel src --presets es2015 --out-dir dist",
-    "watch": "babel -w src  --presets es2015 --out-dir dist"
+    "build": "babel src --presets es2015 --plugins transform-es2015-modules-umd --out-dir dist",
```
Comment (onury) [inline package.json]: This way in DOM, it's exposed as `window.index`, not `window.invert`.

**Review by onury — COMMENTED** [review]
`invert` object is not exposed properly in DOM. It was exported as `window.index` per the filename.

**Comment by onury** [comment]
> It was simpler to just use babel's plugin to achieve UMD.

That's better.

**Pull request merged**